


























74911it [00:52, 1450.79it/s]/usr/local/lib/python3.9/dist-packages/torch/nn/modules/loss.py:922: UserWarning: Using a target size (torch.Size([3500, 3500])) that is different to the input size (torch.Size([])). This will likely lead to incorrect results due to broadcasting. Please ensure they have the same size.
  return F.smooth_l1_loss(input, target, reduction=self.reduction, beta=self.beta)


76092it [00:58, 280.86it/s]/usr/local/lib/python3.9/dist-packages/numpy/core/fromnumeric.py:3432: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/usr/local/lib/python3.9/dist-packages/numpy/core/_methods.py:190: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)











































































































































142620it [05:39, 292.09it/s]Traceback (most recent call last):
  File "/notebooks/dqn-test/breakout.py", line 148, in <module>
    pass
AttributeError: 'FrameStackingAndResizingEnv' object has no attribute 'close'